sdks/python/apache_beam/runners/worker/data_plane.py

```diff
@@ -274,20 +301,47 @@ def inverse(self):
     return self._inverse
 
   def input_elements(self,
-                     instruction_id,  # type: str
-                     unused_expected_transforms=None,  # type: Optional[Collection[str]]
-                     abort_callback=None  # type: Optional[Callable[[], bool]]
-                    ):
-    # type: (...) -> Iterator[beam_fn_api_pb2.Elements.Data]
```

Comment:
It'd be good to not lose the typing information. You can make an alias at the top of the file `DataOrTimers = Union[beam_fn_api_pb2.Elements.Data, beam_fn_api_pb2.Elements.Timer]` to cut down on verbosity, here and elsewhere.